service/models/converters.py

```diff
@@ -44,15 +44,22 @@ def fetch_and_convert_dynamic_interventions(policy_intervention_id, job_id):
 # Used to convert from HMI Intervention Policy -> pyciemss static interventions.
 def convert_static_interventions(interventions: list[HMIIntervention]):
     if not (interventions):
-        return defaultdict(dict)
-    static_interventions: Dict[torch.Tensor, Dict[str, any]] = defaultdict(dict)
+        return defaultdict(dict), defaultdict(dict)
+    static_param_interventions: Dict[torch.Tensor, Dict[str, any]] = defaultdict(dict)
+    static_state_interventions: Dict[torch.Tensor, Dict[str, any]] = defaultdict(dict)
     for inter in interventions:
         for static_inter in inter.static_interventions:
-            time = torch.tensor(float(static_inter.timestep))
-            parameter_name = inter.applied_to
-            value = torch.tensor(float(static_inter.value))
-            static_interventions[time][parameter_name] = value
-    return static_interventions
+            if inter.type == "parameter":
+                time = torch.tensor(float(static_inter.timestep))
+                parameter_name = inter.applied_to
+                value = torch.tensor(float(static_inter.value))
+                static_param_interventions[time][parameter_name] = value
+            if inter.type == "state":
+                time = torch.tensor(float(static_inter.timestep))
+                parameter_name = inter.applied_to
+                value = torch.tensor(float(static_inter.value))
+                static_state_interventions[time][parameter_name] = value
```

Comment:
what's the point of this, dont they do the same thing?

Reply:
Need to filter states into one variable and parameters into another otherwise ciemss gets confused